pkg/feature/cert.go

```diff
@@ -3,13 +3,12 @@ package feature
 import (
 	"bytes"
 	"context"
-	cryptorand "crypto/rand"
+	"crypto/rand"
```

Comment:
is lint complaining with alias ?

Reply:
well no... lint didnot like the usage of `math/rand` so I had to use `crypto/rand` and I thought since `math/rand` was removed it was unnecessary to have an alias 😅

Reply:
for code consistency i am fine with this